### ISSUE TITLE
Menu Component a11y issue

### ISSUE THREAD

**Issue by SedueRey** [primefaces/primevue#78]
Hello,
I'm using primevue's Menu component and, when I check a11y issues from insight I get this error:

`aria-required-parent: Certain ARIA roles must be contained by particular parents  WCAG 1.3.1`

Because generated `<ul>` doesn't have a menu or menubar role although `<li>` items do have  `role="menuitem"`.

Kind regards,
